source/_patterns/03-organisms/navbar/demo/navbars.md

```diff
@@ -0,0 +1,6 @@
+| Name              | Type   | Options | Description                          |
```

Comment:
Does it make sense to include applicable Bootstrap classes in these markdown files? Might be a nice "save you a click" for using this.